pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -0,0 +1,55 @@
+package routefix
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	securityclient "github.com/openshift/client-go/security/clientset/versioned"
+	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/Azure/ARO-RP/pkg/operator/controllers"
+)
+
+//RouteFixReconciler is the controller struct
+type RouteFixReconciler struct {
+	kubernetescli kubernetes.Interface
+	seccli        securityclient.Interface
+	log           *logrus.Entry
+}
+
+//NewReconciler creates a new Reconciler
+func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, seccli securityclient.Interface) *RouteFixReconciler {
+	return &RouteFixReconciler{
+		log:           log,
+		kubernetescli: kubernetescli,
+		seccli:        seccli,
+	}
+}
+
+//Reconcile fixes the daemonset Routefix
+func (r *RouteFixReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
```

Comment:
here, copy/paste:
```
	ctx := context.Background()
	if request.Name != arov1alpha1.SingletonClusterName {
		return reconcile.Result{}, nil
	}

	instance, err := r.arocli.Clusters().Get(ctx, request.Name, metav1.GetOptions{})
	if err != nil {
		return reconcile.Result{}, err
	}
```

That gets you the Cluster object to pick out the ACRName.